src/util/Constants.ts

```diff
@@ -380,4 +397,173 @@ export enum WebhookTypes {
+     */
+    READ_MESSAGE_HISTORY = (1 << 16),
+    /**
+     * Allows for using the `@everyone` tag to notify all users in a channel, and the `@here` tag to notify all online users in a channel.
+     */
+    MENTION_EVERYONE = (1 << 17),
+    /**
+     * Allows the usage of custom emojis from other servers.
+     */
+    USE_EXTERNAL_EMOJIS = (1 << 18),
+    /**
+     * Allows for viewing guild insights.
+     */
+    VIEW_GUILD_INSIGHTS = (1 << 19),
+    /**
+     * Allows for joining a voice channel.
+     */
+    CONNECT = (1 << 20),
+    /**
+     * Allows for speaking in a voice channel.
+     */
+    SPEAK = (1 << 21),
+    /**
+     * Allows for muting of members in a voice channel.
+     */
+    MUTE_MEMBERS = (1 << 22),
+    /**
+     * Allows for deafening of members in a voice channel.
+     */
+    DEAFEN_MEMBERS = (1 << 23),
+    /**
+     * Allows for moving of members between voice channels or disconnecting.
+     */
+    MOVE_MEMBERS = (1 << 24),
+    /**
+     * Allows for using voice-activity-detection in a voice channel.
+     */
+    USE_VAD = (1 << 25),
+    /**
+     * Allows for modification of own nickname.
+     */
+    CHANGE_NICKNAME = (1 << 26),
+    /**
+     * Allows for modification of other users nicknames.
+     */
+    MANAGE_NICKNAMES = (1 << 27),
+    /**
+     * Allows management and editing of roles.
+     */
+    MANAGE_ROLES = (1 << 28),
+    /**
+     * Allows management and editing of webhooks
+     */
+    MANAGE_WEBHOOKS = (1 << 29),
+    /**
+     * Allows management and editing of emojis and stickers.
+     */
+    MANAGE_EMOJIS_AND_STICKERS = (1 << 30),
+    /**
+     * Allows members to use application commands, including slash commands and context menu commands.
+     */
+    USE_APPLICATION_COMMANDS = (1 << 31),
+    /**
+     * Allows for requesting to speak in stage channels.
+     */
+    REQUEST_TO_SPEAK = (1 << 32),
+    /**
+     * Allows for creating, editing, and deleting scheduled events.
+     */
+    MANAGE_EVENTS = (1 << 33),
+    /**
+     * Allows for deleting and archiving threads, and viewing all private threads.
+     */
+    MANAGE_THREADS = (1 << 34),
+    /**
+     * Allows for creating threads.
+     */
+    CREATE_PUBLIC_THREADS = (1 << 35),
+    /**
+     * Allows for creating private threads.
+     */
+    CREATE_PRIVATE_THREADS = (1 << 36),
+    /**
+     * Allows the usage of custom stickers from other servers.
+     */
+    USE_EXTERNAL_STICKERS = (1 << 37),
+    /**
+     * Allows for sending messages in threads.
+     */
+    SEND_MESSAGES_IN_THREADS = (1 << 38),
+    /**
+     * Allows for launching activities (applications with the `EMBEDDED` flag) in a voice channel.
+     */
+    START_EMBEDDED_ACTIVITIES = (1 << 39),
+    /**
+     * Allows for timing out users to prevent them from sending or reacting to messages in chat and threads, and from speaking in voice and stage channels.
+     */
+    MODERATE_MEMBERS = (1 << 40),
+}
```

Comment:
Can we use bigints for this?

Reply:
> Can we use bigints for this?

Done in [here](https://github.com/BirbJS/Birb/pull/38/commits/20964149a949f3a0be87238700f799c14a26734e).